### ISSUE TITLE
Filenames too long

### ISSUE THREAD

**Issue by johnrizzo1** [IdentityServer/IdentityServer4.Samples#179]
Looks like we have some filenames that have gotten too long for windows.  I'm running windows 10 and here is my git checkout

λ  git clone https://github.com/IdentityServer/IdentityServer4.Samples.git
Cloning into 'IdentityServer4.Samples'...
remote: Counting objects: 3916, done.
remote: Total 3916 (delta 0), reused 0 (delta 0), pack-reused 3916
Receiving objects: 100% (3916/3916), 6.23 MiB | 0 bytes/s, done.
Resolving deltas: 100% (2612/2612), done.
error: unable to create file Quickstarts/8_EntityFrameworkStorage/src/QuickstartIdentityServer/Data/Migrations/IdentityServer/ConfigurationDb/20170319194441_InitialIdentityServerConfigurationDbMigration.Designer.cs: Filename too long
error: unable to create file Quickstarts/8_EntityFrameworkStorage/src/QuickstartIdentityServer/Data/Migrations/IdentityServer/PersistedGrantDb/20170319194432_InitialIdentityServerPersistedGrantDbMigration.Designer.cs: Filename too long
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'

**Comment by brockallen** [comment]
perhaps clone from "c:\code"? my path where i work from is "C:\ballen\github\identity\IdSvr4\Samples"

**Comment by mccarty82** [comment]
I am suggesting to try "Long Path Tool" program.

**Comment by smoses2** [comment]
I've cloned this repo a few times in the last couple of years and this is the first time I ran into this "filename too long" problem with this repo.  My install directory structure is already pretty short and would rather not run another tool (e.g. Long Path Tool).

I ran across a suggestion to put in the .gitconfig file:
[core]
longpaths = true

Another option - wonder if it would be possible to shorten some of the names/directories in this repo with abbreviations.  

Can't thank you enough for identityserver - phenomenal opus.
